static/css/style.css

```diff
@@ -2306,7 +2314,7 @@ ul.messages {
   margin: auto;
   outline: none;
   border: none;
-  background: var(--lightestGray);
+  background: rgba(0, 0, 0, 0.5);
```

Comment:
This color swap makes the dismiss button more visible. 

<img width="39" alt="image" src="https://user-images.githubusercontent.com/2692333/117883875-92ddf380-b271-11eb-8025-2df7b6626300.png">

<img width="44" alt="image" src="https://user-images.githubusercontent.com/2692333/117883901-9a9d9800-b271-11eb-85ad-57ef04a5be85.png">